xcube_sh/config.py

```diff
@@ -74,6 +74,16 @@ class CubeConfig:
     :param four_d: If variables should appear as forth dimension rather
         than separate arrays.
     :param exception_type: The type of exception to be raised on error
+    :param processing_kwargs: Processing Keywords Arguments to be passed to Sentinel Hub
+    such as for S1GRD -> {
+                    "orthorectify": "false",
+                    "backCoeff": "GAMMA0_ELLIPSOID",
+                    "speckleFilter": {
+                        "type": "LEE",
+                        "windowSizeX": 5,
+                        "windowSizeY": 5
+                    }
+    defaults to None
```

Comment:
```suggestion
    :param processing_kwargs: Processing Keywords Arguments to be passed to Sentinel Hub
        such as for S1GRD -> ``{
                    "orthorectify": "false",
                    "backCoeff": "GAMMA0_ELLIPSOID",
                    "speckleFilter": {
                        "type": "LEE",
                        "windowSizeX": 5,
                        "windowSizeY": 5
                    }``.
        Defaults to ``None``.
```

Reply:
Thanks @forman ! 👍 

I have applied the changes regarding the docstring and updated the `CHANGES.md` .